src/main/java/org/folio/rest/core/RestClient.java

```diff
@@ -28,12 +30,11 @@ public <T> CompletableFuture<T> getById(String baseEndpoint, String id, RequestC
     return get(requestEntry, requestContext, responseType);
   }
 
-  public <S> CompletableFuture<S> get(RequestEntry requestEntry, RequestContext requestContext, Class<S> responseType) {
+  private <S> CompletableFuture<S> get(RequestEntry requestEntry, RequestContext requestContext, Function<JsonObject, S> responseHandler) {
```

Comment:
If it's used only in unit tests - lets delete it, probably it was some similar migration previously that called mod-orders-storage, but now this migration is deleted and support code also would be deleted.
Remaining it as business code that can be used by some business usage is also not applicable here, because mod-invoice-storage-should not invoke mod-orders-storage, instead based on current architecture these invocations should be done on business modules like mod-orders/mod-invoice